--- a/vulkan-headers/CMakeLists.txt
+++ b/vulkan-headers/CMakeLists.txt
@@ -69,7 +69,7 @@ if (PROJECT_IS_TOP_LEVEL)
     # Install registry files
     install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/registry" DESTINATION ${VLK_REGISTRY_DIR} USE_SOURCE_PERMISSIONS)
     # Where *.cmake files will be installed
-    set(cmake_files_install_dir ${CMAKE_INSTALL_DATADIR}/cmake/VulkanHeaders/)
+    set(cmake_files_install_dir ${CMAKE_INSTALL_LIBDIR}/cmake/VulkanHeaders/)
 
     # Set EXPORT_NAME for consistency with established names. The CMake generated ones won't work.
     set_target_properties(Vulkan-Headers PROPERTIES EXPORT_NAME "Headers")
